Platform/ArrowlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -831,6 +831,9 @@ UpdateFspConfig (
         FspsConfig->PcieGen5EqPh3NoOfPresetOrCoeff[Index] = 0x3;
         FspsConfig->PcieGen5EqPh3Preset2List[Index] = 0x7;
         FspsConfig->PcieGen5EqPh3Preset3List[Index] = 0x0;
+      }
+      UINT8 PciePortNum = sizeof (FspsConfig->PcieEnablePeerMemoryWrite);
```

Comment:
@ongeelim pls update the variable name a generic one, and declare it at beginning of the function for consistent. thanks

Reply:
Updated by removing that variable and just use sizeof function in the for statement.